webpack-5/README.md

```diff
@@ -0,0 +1,90 @@
+# cesium-webpack-example
+
+A minimal recommended setup for an applications using [Cesium](https://cesium.com) with [Webpack](https://webpack.js.org/concepts/).
```

Comment:
```suggestion
A minimal recommended setup for an applications using [Cesium](https://cesium.com) with [Webpack 5](https://webpack.js.org/concepts/).
```

Reply:
Should there be a version of this README for the Webpack 4 directory?

Reply:
Yes, I thought I had copied it over already but apparently missed it, updated